interface/batchcom/batchPhoneList.php

```diff
@@ -6,42 +6,45 @@
 // create a list for phone calls
 // menu for fields could be added in the future
 
+include_once("../globals.php");
+use OpenEMR\Core\Header;
+
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="batchcom.css" type="text/css">
```

Comment:
this is a "legacy" script so isn't using xlt/xla yet.
xl() does basic translation

text() does html escaping outside html attributes
attr() does html escaping inside html attributes

xlt() is xl() + text()
xla is xl() + attr()

The legacy scripts use xl(,'e') which is same as echo xl() . We stopped doing this because this does not allow html escaping. I usually don't have developers fix this stuff when they work on something, but it seemed like a good opportunity for you to learn this stuff since you are basically re-optimizing these scripts.